### PR TITLE
To do list[ add & remove] project

### PR DESCRIPTION
**Pull request by Abbas-Aljuaifari**
In this project, I worked on the To do list project finalized the structure part, and added more functionality to the website also worked with `webpack` and bundle.js and set my files with `ES6 Modular,` In the final part of the project i will remodify the application and make it into a fully functional website. 

In this project, I worked on and modified the following.

 

- [X] Worked on the `index.HTML` file and apply the best practice methods and added all the requirements.
- [X] Worked on the `style.Css` file and apply the best practice methods and added all the requirements.
- [X] Worked on the `index.js` file and apply the best practice methods and added all the requirements.
- [X] Worked on the `Webpack` and apply the best practice methods and added all the requirements.
- [X] Worked on the `Modular Directory` and apply the best practice methods and added all the requirements.
- [X] Worked on the `Linters files` and checked all errors.

### CODE REVIEW

File: src/modules/taskList.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    this.tasks.push(newTask);
+    this.saveTasks();
+  }
+
+  deleteTask(index) {
+    this.tasks.splice(index, 1);
+    this.updateIndexes();
+    this.saveTasks();
+  }
+
```
Comment (ghost) [inline src/modules/taskList.js]: - [ ] You have done a fantastic job on this milestone. However, it is recommended in the project's requirements that all the properties should appear including the `index` after adding a task. However, in your case, the index property only appears after deleting a task. Kindly rectify that.

### After adding tasks

image

### After Deleting one task

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Abbas-Aljuaifari , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ App is working as expected👍 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aamirkhan2478 — APPROVED** [review]
Hi @Abbas-Aljuaifari , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibizugbe** [comment]
Hi @Abbas-Aljuaifari 👋🏼👋🏼👋🏼, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] Project has met the requirements for the milestone👍🏼.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**